src/utils/converter.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable  */
```

Comment:
- [ ] Please avoid disabling eslint rule for an entire module. Kindly remove this line.